README.md

```diff
@@ -1,24 +1,193 @@
-# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <br/>
 
-Things you may want to cover:
+  <h3><b>Budget App</b></h3>
 
-* Ruby version
+</div>
 
-* System dependencies
+<!-- TABLE OF CONTENTS -->
 
-* Configuration
+# 📗 Table of Contents
 
-* Database creation
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-* Database initialization
+<!-- PROJECT DESCRIPTION -->
 
-* How to run the test suite
+# 📖 [Budget App] <a name="about-project"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+**[Budget App]**  a mobile web application designed for budget management. In this application, users can efficiently oversee their financial transactions by categorizing them. By doing so, they can gain insight into their spending patterns, allowing them to track their expenses and understand where their money is allocated. This project not only showcases the use of Ruby on Rails but also serves as a practical tool for individuals to maintain control over their finances.
 
-* Deployment instructions
+## 🛠 Built With <a name="built-with"></a>
 
-* ...
+- Ruby on Rails
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Test</summary>
+  <ul>
+    <li>RSPEC</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>PostgreSQL</li>
+  </ul>
+</details>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](Comming soon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Presentation Video -->
+## 🎥 Presentation Video <a name="presentation-video"></a>
+
+- [Presentation Video Link](comming soon)
+
```

Comment:
- [ ] Kindly deploy your app and provide its live demo link on the readme documentation.

- [ ] Kindly make a presentation of your app, preferably on Loom, demonstrating features of the app. The video should not be more than 5 minutes long and kindly also highlight code or feature you find interesting.